contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViewConstants.java

```diff
@@ -502,6 +513,71 @@
           COUNT,
           Arrays.asList(GRPC_CLIENT_METHOD));
 
+  /**
+   * {@link View} for client retries per call.
+   *
+   * @since 0.28
+   */
+  public static final View GRPC_CLIENT_RETRIES_PER_CALL_VIEW =
+      View.create(
+          View.Name.create("grpc.io/client/retries_per_call"),
+          "Number of client retries per call",
+          GRPC_CLIENT_TRANSPARENT_RETRIES_PER_CALL,
```

Comment:
should this be GRPC_CLIENT_RETRIES_PER_CALL ?

Reply:
yes great catch, thanks @asafdav2 - updated